nuvlaedge/peripherals/network/__main__.py

```diff
@@ -1,4 +1,5 @@
 from nuvlaedge.peripherals.network import main
+import asyncio
 
-if __name__ == '__main__':
-    main()
+def entry():
```

Comment:
Could we keep the `if __name__ == '__main__':` check and entry() call in the __main__ file? That allows for the given package to be called from the cmd without the need of installing it into /usr/local/bin.

I also think the entry() function might fit better in __init__.py and then called from __main__ to keep the above mentioned functionality

Reply:
Agree with your viewpoint. I have changed accordingly for all peripherals. Have tested on the nuvlaedge in rpi.